packages/terra-pills/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "terra-pills",
+  "main": "lib/index.js",
+  "version": "0.0.0",
+  "description": "The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-pills"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-pills",
+    "Pills",
+    "Pill",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "files": [
+    "*.log",
+    "target",
+    "reports",
+    "tests/**/__snapshots__/",
+    "aggregated-translations",
+    "dev-site-config",
+    "screenshots",
+    "errorScreenshots",
+    "generated-themes"
```

Comment:
files is the list of files to include in the package, vs .npmignore which is the list of files to exclude. You'll want to change this to something like: https://github.com/cerner/terra-framework/blob/main/packages/terra-framework-docs/package.json#L28-L30

there are a couple of files included by default so you don't have to worry about those: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

Reply:
This still needs fixed or the package won't be useable when released.

Reply:
changed here ee1622133ac54f2deacf7efbaa829413121f73e6